src/packageManager/Package.ts

```diff
@@ -3,15 +3,17 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-export interface Package {
+export interface IPackage {
     description: string;
     url: string;
     fallbackUrl?: string;
-    installPath?: string;
     platforms: string[];
     architectures: string[];
-    binaries: string[];
     platformId?: string;
-    // Path to use to test if the package has already been installed
+}
+
+export interface Package extends IPackage{
```

Comment:
`RelativePathPackage`?

Reply:
I am not sure we should do that. The paths here can be absolute or relative there is no specific restriction that we are imposing right(like isRelative(path) or something). Thoughts ?